src/snowflake/connector/converter.py

```diff
@@ -366,8 +366,8 @@ def _nonetype_to_snowflake_bindings(self, *_) -> None:
         return None
 
     def _date_to_snowflake_bindings(self, _, value: date) -> str:
-        # milliseconds
-        return _convert_date_to_epoch_milliseconds(value)
+        # we are binding "TEXT" value for DATE, check function _adjust_bind_type
+        return value.strftime("%Y-%m-%d")
```

Comment:
minor suggestion: We could rely [on the existing `datetime.date#isoformat(…)` method](https://github.com/python/cpython/blob/023c51d9d8e2fd91069eea2bf12e373f1c71e9d2/Lib/datetime.py#L1046-L1055) which is more explicit about the widths used, just in case there's gotchas with early AD/late BC dates?

Reply:
thanks for the suggestion, applied your suggestion